word_guess.rb

```diff
@@ -0,0 +1,193 @@
+class WordGuess
+  def initialize
+    @word_array = ["DOG", "CAT", "PET", "HUT"]
+    @word = @word_array.sample              #chooses a random word
+    @letter_array = @word.split("")         #splits the random word's letter into an array
+    @dash_array = [ " _ ", " _ ", " _ "]
+    @letters_guessed = []
+    @rose_array = ["(@)", "(@)", "(@)", "(@)", "(@)"]
+    #@user_letter = user_letter
+
+    puts @word       # Remove this test before final push.
+
+
+                    # \ here is start of an escape sequence with n being a new line, so \ \ is an actual backslash!!!
+    puts "\n
+    #{@rose_array.join}
+      \\ \\ | / /
+        _\\|/_
+       |_____|
+        |   |
+        |___| "         # \ is an escape sequence...\ is starting an escape sequence that escapes to a \
+                        #this prints out the dash array
+    puts @dash_array.join
+
+    puts "\nPlease choose a letter and press enter!"
+    @user_letter = gets.chomp.upcase
+    puts "You selected #{@user_letter}."
+    @letters_guessed << @user_letter    #Adds the letter user guessed to the array
+    include_in_word
+  end
+
+def include_in_word
+    if @letter_array.include?(@user_letter) == true
+      puts "You guessed correctly!"
+      #Enter code that changes the dash to the letter the user guessed right heeere
+        if @word == "PET"
+          word_pet
+        elsif @word == "HUT"
+          word_hut
+        elsif @word == "CAT"
+          word_cat
+        elsif @word == "DOG"
+          word_dog
+        end
+
+    else
+      puts "#{@user_letter} is not included in the word!"
+      @rose_array.delete_at(0)
+      puts "\n
```

Comment:
It seems like you do this output of the flower pot ASCII art in two different places. This would be a good place to create a `draw_flower` method that would take care of this drawing for you so you don't have this code duplicated.